tests/attacks/test_copycat_cnn.py

```diff
@@ -20,24 +20,30 @@
 import logging
 import unittest
 
-import tensorflow as tf
-import numpy as np
 import keras
 import keras.backend as k
-from keras.models import Sequential
-from keras.layers import Dense, Flatten, Conv2D, MaxPooling2D
+import numpy as np
+import tensorflow as tf
 import torch
 import torch.nn as nn
 import torch.optim as optim
+from keras.layers import Conv2D, Dense, Flatten, MaxPooling2D
+from keras.models import Sequential
+from tests.utils import (
+    TestBase,
+    get_image_classifier_kr,
+    get_image_classifier_pt,
+    get_image_classifier_tf,
+    get_tabular_classifier_kr,
+    get_tabular_classifier_pt,
+    get_tabular_classifier_tf,
+    master_seed,
+)
```

Comment:
I think we should keep the imports from `tests` after the imports from `art` because they are not third-party and to keep it consistent across all tests.